x-pack/plugins/lens/public/xy_visualization/types.ts

```diff
@@ -424,7 +424,7 @@ export const visualizationTypes: VisualizationType[] = [
     id: 'bar_horizontal',
     icon: LensIconChartBarHorizontal,
     label: i18n.translate('xpack.lens.xyVisualization.barHorizontalLabel', {
-      defaultMessage: 'Horizontal bar',
+      defaultMessage: 'H. Bar',
```

Comment:
Copy-wise, is there a reason we sometimes write all starting from capital letter and sometimes not?
`H. **B**ar` but `Percentage **b**ar`
`Mixed H. **b**ar` but `H. **B**ar` etc

Reply:
Ok, I just saw Caroline's comment, I guess it's rules of English grammar 😅